tap_github/repository_streams.py

```diff
@@ -404,14 +404,14 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
         th.Property(
             "target_repo",
             th.ObjectType(
-                th.Property("id", th.StringType),
+                th.Property("id", th.IntegerType),
```

Comment:
It seems odd that the parent repo id would be a string and the target repo id an integer?